Ginger/Ginger/Actions/_Common/ActUIElementLib/ActUIElementEditPage.xaml.cs

```diff
@@ -629,5 +676,18 @@ private string GetPomElementOptionalValuesDefaultValue(ePomElementValuesType val
             }
             return defaultValue;
         }
+
+        private void XWidgetsElementType_Click(object sender, RoutedEventArgs e)
+        {
+            if(xWidgetsElementType.IsChecked == true)
```

Comment:
if you will bind this check box then don't this if else to explicitly add input value